lib/main.js

```diff
@@ -341,19 +449,45 @@ exports.main = function () {
     contentScript: 'self.on("click", self.postMessage)',
     onMessage: launchConfigTab
   });
+
   checkUpdate();
   setUpTopMenu(initSubMenus());
+
+  // Adding a simple playlist.
+  plists.playlist_id = plists.add_playlist();
+  plists.playlist_id = plists.add_playlist();
```

Comment:
After testing, the index problem seems to come from Kodi, playlists indexes start at 1, not zero. I will make a change, still, to fix this, because it is not really logic to add two playlists in order to have one ...